gamla/functional_generic.py

```diff
@@ -204,7 +226,22 @@ def pipe(val, *funcs):
     return compose_left(*funcs)(val)
 
 
+#:  Map an iterable using a function, return `True` iff all mapped values are `True`-ish.
+#:
+#:    >>> f = allmap(lambda x: x % 2 == 0)
+#:    >>> f([1, 2, 3, 4, 5])
+#:    False
+#:    >>> f([2, 4, 6, 8, 10])
+#:    True
 allmap = compose(after(all), curried_map)
+
+#:  Map an iterable using a function, return `True` if at least one mapped value is `True`-ish.
+#:  Note: evaluation is lazy, i.e. returns on first `True`.
+#:    >>> f = anymap(lambda x: x % 2 == 0)
```

Comment:
try to be consistent with the newline between

also i'm not sure if you need the indent below